index.html

```diff
@@ -0,0 +1,162 @@
+                    <i class="fas fa-video size"></i>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main id="grid-container-main">
+        <div id="section-container">
+            <section id="section-container-main">
+                <div>
+                    <iframe src="https://www.youtube.com/embed/WNOBl1mtsMY"
+                        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                        allowfullscreen></iframe>
+                </div>
+                <h1>Cute and Funny Dog Compilation</h1>
+                <div id="grid-container-video-main-info">
+                    <p>1,340,783 views • 18 Dec 2018</p>
+                    <div id="grid-container-video-main-info-extra">
+                        <i class="fa fa-thumbs-up" aria-hidden="true"> 100K</i>
+                        <i class="fa fa-thumbs-down" aria-hidden="true"> 110</i>
+                        <i class="fa fa-share"> Share</i>
+                    </div>
+                </div>
+            </section>
+            <hr>
+            <section id="grid-container-video-additional-info">
+                <div id="video-owner">
+                    <i class="fab fa-adn"></i>
+                </div>
+                <div id="video-information">
+                    <h2>Animal World</h2>
+                    <h3>100K Subscribers</h3>
+                    <p id="video-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Non voluptates
+                        exercitationem aliquid quod sit reiciendis voluptatibus ullam, magni optio ea laudantium totam
+                        beatae. Iusto autem repudiandae rerum rem, quae animi!</p>
+                    <p id="video-description-more">SHOW MORE</p>
+                </div>
+                <div id="subscribe-button">
+                    <button>SUBSCRIBE</button>
+                </div>
+            </section>
+        </div>
+        <aside>
+            <section class="grid-aside">
+                <div>
+                    <h3>Up next</h3>
+                </div>
+                <div id="thumbnail">
+                    <span>AUTOPLAY</span>
+                    <i class="fa fa-toggle-on fa-lg" aria-hidden="true"></i>
+                </div>
+            </section>
+            <section id="first-thumbnail" class="grid-aside">
+                <div class="sidebar-image cat-one">
+                    <p class="sidebar-caption">14:50</p>
+                </div>
+                <div class="thumbnail-description">
+                    <h4>Definitely Cute</h4>
+                    <p class="thumbnail-info">Lorem, ipsum <span><i class="fa fa-check-circle"
+                                aria-hidden="true"></i></span></p>
+                    <p>1.7M views • 2 years ago</p>
+                </div>
+            </section>
+            <hr>
+            <section id="second-thumbnail" class="grid-aside">
```

Comment:
- [ ] Pay more atention to the fonts here as well

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/90419731-5d62fb80-e08d-11ea-9713-1090f0fda7e3.png)|![image](https://user-images.githubusercontent.com/61479648/90419764-694ebd80-e08d-11ea-8596-af9239cf5692.png) |